servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/catalog/CatalogTests.java

```diff
@@ -173,9 +174,7 @@ protected boolean supportsNamesWithDot() {
     return true;
   }
 
-  protected String temporaryLocation() {
-    return "file:/tmp/ns/table";
-  }
+  protected abstract String temporaryLocation();
```

Comment:
Better leave the original one. The intent of `CatalogTests.java` + `ViewCatalogTests.java` being here is to adopt those for Iceberg REST and later contribute the changes back to Iceberg.